macOS/LocalPackages/SubscriptionUI/Sources/SubscriptionUI/SubscriptionAccessView/SubscriptionAccessViewModel.swift

```diff
@@ -44,23 +44,19 @@ public final class SubscriptionAccessViewModel {
 
     public func handleEmailAction() {
         actionHandlers.openActivateViaEmailURL()
-        actionHandlers.uiActionHandler(.activateSubscriptionViaEmailClick)
```

Comment:
Passing `uiActionHandler` that was to handle many more than these 2 actions here was unnecessary. Instead the side effect of `uiActionHandler` calls are internally wrapped in 2 actions from the `actionHandlers`